packages/components/src/divider/types.ts

```diff
@@ -2,7 +2,7 @@
  * External dependencies
  */
 // eslint-disable-next-line no-restricted-imports
-import type { SeparatorProps } from 'reakit';
+import type { SeparatorProps } from '@ariakit/react';
```

Comment:
It looks like the `SeparatorProps` from `ariakit` include the `as` and `render` props, which we don't want to expose.

We can easily fix this by adding `'as'` and `'render'` to the list of omitted props on line 14, but it would be also interesting to know if there;s any particular reason why previously we were not picking up the `as` prop from `SeparatorProps`, and now we are.

Reply:
I applied your suggestions via [7d0b7ce](https://github.com/WordPress/gutenberg/pull/55622/commits/7d0b7cea045642d6d91449643645bb00787d19c3) – I'm a bit unsure about the rationale you mentioned.

Reply:
Apologies in case I wasn't clear. What I was trying to say is that I was curious to know the reason why previously, when using `reakit` types, the `as` prop was not being picked up from `SeparatorProps`. I guess the answer is simply because `SeparatorProps` did not include the `as` prop .